faunadb/_json.py

```diff
@@ -10,7 +11,10 @@ def parse_json(json_string):
   Parses a JSON string into python values.
   Also parses :any:`Ref`, :any:`Set`, :any:`FaunaTime`, and :class:`date`.
   """
-  return loads(json_string, object_hook=_parse_json_hook)
+  try:
+    return loads(json_string, object_hook=_parse_json_hook)
+  except ValueError:
+    raise InvalidResponse("Bad json: %s" % json_string)
```

Comment:
I'm using InvalidResponse for several different cases. Should I create a `class InvalidJson(InvalidResponse)` for this specific case?